.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip wheel
+    - name: Build the package
+      id: build
+      run: |
+           python setup.py sdist bdist_wheel --universal
```

I am not a Python setuptools expert, but I am fairly sure we don't need `sdist` and `--universal` here. I think we use those on some of our libraries because we upload them to PyPI, but here, the previous Travis job was just running `python setup.py bdist_wheel`.

```suggestion
           python setup.py bdist_wheel
```

---

Done